stdlib/public/Concurrency/CMakeLists.txt

```diff
@@ -291,8 +292,37 @@ if(SWIFT_SHOULD_BUILD_EMBEDDED_STDLIB AND SWIFT_SHOULD_BUILD_EMBEDDED_CONCURRENC
       PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
       )
     set_property(TARGET embedded-concurrency-${mod} PROPERTY OSX_ARCHITECTURES "${arch}")
-
     add_dependencies(embedded-concurrency embedded-concurrency-${mod})
+
+    # lib/swift/embedded/<triple>/libswift_ConcurrencyDefaultExecutor.a
+    add_swift_target_library_single(
+      embedded-concurrency-default-executor-${mod}
+      swift_ConcurrencyDefaultExecutor
+      STATIC
+      IS_FRAGILE
+
+      CooperativeGlobalExecutor.cpp
+
+      C_COMPILE_FLAGS ${extra_c_compile_flags}
+      MODULE_DIR "${CMAKE_BINARY_DIR}/lib/swift/embedded"
+      SDK "embedded"
+      ARCHITECTURE "${mod}"
+      DEPENDS embedded-stdlib-${mod}
+      INSTALL_IN_COMPONENT stdlib
+      )
+    swift_install_in_component(
+      TARGETS embedded-concurrency-default-executor-${mod}
+      DESTINATION "lib/swift/embedded/${mod}"
+      COMPONENT "stdlib"
+      )
+    swift_install_in_component(
+      FILES "${SWIFTLIB_DIR}/embedded/${mod}/libswift_ConcurrencyDefaultExecutor.a"
+      DESTINATION "lib/swift/embedded/${mod}/"
+      COMPONENT "stdlib"
+      PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
```

Comment:
`.a` files don't need or want execute permission. Most systems don't even need it for _dynamic_ libraries, but it's definitely not a thing for static libraries.

Reply:
Agree in principle, but see <https://github.com/swiftlang/swift/pull/73599> -- there is some pre-existing toolchain packaging logic that requires the +x permission even on .a libraries. Sounds like something worth looking into, I'll try to do that later.

Reply:
It's minor so definitely not needed for this PR. Would be nice to fix that though, as it's pretty weird having the `x` bits set for these.